docs/integrate/dbeaver/index.md

```diff
@@ -0,0 +1,85 @@
+(dbeaver)=
+# DBeaver
+
+:::{include} /_include/links.md
+:::
+
+```{div}
+:style: "float: right; margin-left: 0.5em"
+[![](https://upload.wikimedia.org/wikipedia/commons/thumb/b/b5/DBeaver_logo.svg/512px-DBeaver_logo.svg.png){w=120px}](https://dbeaver.io/)
+```
+
+[DBeaver] is a multipurpose cross-platform database tool for developers,
+database administrators, analysts, and everyone working with data.
+
+It is available as an open-source version _DBeaver Community_ and
+as a commercial version _DBeaver PRO_.
+
+
+## Connect
+
+::::{grid} 2
+
+:::{grid-item}
+Please specify database URL and credentials of your CrateDB cluster.
+For connecting to CrateDB, the standard [PostgreSQL JDBC Driver]
+will be used.
+
+When connecting to [CrateDB Self-Managed] on localhost,
+for evaluation purposes, use:
+```
+jdbc:postgresql://localhost:5432/crate
+```
+
+When connecting to [CrateDB Cloud], use:
+```
+jdbc:postgresql://<clustername>.cratedb.net:5432/crate
+```
+:::
+:::{grid-item}
+![Image](https://github.com/user-attachments/assets/630fcc7c-21c5-4070-be72-e38041c19d8e){w=480px}
+:::
+
+::::
+
+
+## Usage
+Use the tree menu on the left-hand pane to navigate to the `doc` schema and
+its tables. Navigate to the Data tab to browse your table data.
+
+![Image](https://cratedb.com/hs-fs/hubfs/Screen-Shot-2019-04-05-at-17.15.05.png?width=1600&name=Screen-Shot-2019-04-05-at-17.15.05.png){h=240px}
+![Image](https://cratedb.com/hs-fs/hubfs/Screen-Shot-2019-04-05-at-17.15.13.png?width=1600&name=Screen-Shot-2019-04-05-at-17.15.13.png){h=240px}
+
+
+## Learn
+
+:::{rubric} Tutorials
+:::
+- [Blog: Use CrateDB With DBeaver]
+
+:::{rubric} Product
+:::
+- [CrateDB and DBeaver]
+
+:::{rubric} Notes
+:::
+:::{note}
+A few data types of CrateDB need special considerations.
+- `ARRAY` types are recognised as such in reads, also work for inserts through the GUI.
+  They need to use curly brackets syntax `{1,2}` instead of `[1,2]`.
+- `OBJECT` types are seen as string, and are ok to insert via GUI.
+- `GEO_POINT` types are seen as `STRING`, for insert through GUI, please use parentheses.
+- `GEO_SHAPE` types are seen as `STRING`, are ok to insert via GUI.
+- `FLOAT_VECTOR` types are seen as `ARRAY` on read, and can be inserted
+  using the GUI with same considerations as `ARRAY`s.
+:::
```

Comment:
Thanks for providing this information per https://github.com/crate/crate-clients-tools/issues/206#issuecomment-2535188656, @hlcianfagna.

The same outcome has also been observed here, when verifying vanilla PostgreSQL connectivity support for CrateDB.
- https://github.com/crate/cratedb-examples/pull/772

Please note this means it is just **basic support** that is provided here, because the standard driver does not support many special data types of CrateDB well.